controllers/module_reconciler_test.go

```diff
@@ -27,430 +29,582 @@ const (
 	namespace = "namespace"
 )
 
-var _ = Describe("ModuleReconciler", func() {
-	Describe("Reconcile", func() {
```

Comment:
Didn't we agree to wrap all method-level `Describe` inside one `Describe` per type? cc @ybettan

Reply:
IIRC we agreed that we can use describe per function being tested. No need to use describe for module, since the file its defines which file/module we are testing.

Reply:
+1 for @yevgeny-shnaidman